ansible_rulebook/common.py

```diff
@@ -0,0 +1,29 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import Dict, List
+
+from ansible_rulebook.rule_types import RuleSet
+
+
+@dataclass
+class ActivationPayload:
+    rulesets: List[RuleSet] = field(default_factory=list)
```

Comment:
Why is this a list of rulesets and not the whole rulebook?

Reply:
List of rulesets is the content of the whole rulebook

Reply:
@Alex-Izquierdo @hsong-rh That was just a bad class name on my part. I wanted a data class that can share the arguments if they were coming from websocket or the command lines. I have renamed it to StartupArgs and the ruleset that you mention is the parsed rulesets from a passed in rulebook. In case of command line we get the file name which is parsed and loaded into this RuleSet object, in case of websocket we get the entire ruleset blob which is parsed and then used.